solutions/standard/variables.tf

```diff
@@ -115,3 +115,29 @@ variable "keys" {
   description = "A list of key ring objects each containing one or more key objects. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/tree/main/solutions/standard/DA-keys.md)."
   default     = []
 }
+
+##############################################################
+# Context-based restriction (CBR)
+##############################################################
+
+variable "cbr_rules" {
+  type = list(object({
+    description = string
+    account_id  = string
+    rule_contexts = list(object({
+      attributes = optional(list(object({
+        name  = string
+        value = string
+    }))) }))
+    enforcement_mode = string
+    operations = optional(list(object({
+      api_types = list(object({
+        api_type_id = string
+      }))
+    })))
+  }))
+  description = "(Optional, list) List of context-based restrictions rules to create. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/tree/main/solutions/standard/DA-types.md)"
+  default     = []
+  # Validation happens in the rule module
```

Comment:
I think...we can remove this `# Validation happens in the rule module`

Reply:
done https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/pull/594/commits/67a2aa1c5119d9bbaed014260a89ecfed7c12f95